### PR TITLE
adds template page for pages

### LINKED ISSUE #26: Add additional information to generate-page
- link to "Routing in next"
- link to App Extension navigation setup

### PR DESCRIPTION
**Pull request by katiedavis**
Adds a nice template when people create a page. I didn't want to add too much stuff, as most people will just delete it automatically, but I wanted it to help the user a bit more and look nicer.

Closes #26 
Screen Shot 2019-05-29 at 3 54 16 PM

### CODE REVIEW

File: scripts/page/page-template.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const { toPascalCase } = require("../utilities");
+
+const createPageTemplate = handle => {
+  const componentName = toPascalCase(handle);
+  return `import { Heading, Page } from "@shopify/polaris";
+
+const ${componentName} = () => (
+  <Page>
+    <Heading>You created a new page called ${componentName}</Heading>
+    <p>You can visit this page by appending "/${componentName}" to your URL</p>
```
Comment (nwtn) [inline scripts/page/page-template.js]: what about also something like `you can edit this page in #{file_path}`?

File: scripts/page/page-template.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const { toPascalCase } = require("../utilities");
+
+const createPageTemplate = handle => {
+  const componentName = toPascalCase(handle);
+  return `import { Heading, Page } from "@shopify/polaris";
```
Comment (nwtn) [inline scripts/page/page-template.js]: are you missing a backtick here?

Comment (katiedavis) [reply scripts/page/page-template.js]: no no it's one giant block :/

File: scripts/page/page-template.js (hunk 1 above)
Comment (katiedavis) [reply scripts/page/page-template.js]: Oh good one!

File: scripts/page/page-template.js (hunk 2 above)
Comment (nwtn) [reply scripts/page/page-template.js]: oh durr ya ok

**Pull request merged**